src/instance/InstanceController.ts

```diff
@@ -42,4 +42,37 @@ export class InstanceController {
 
     await this.instanceService.deleteInstance(instanceId);
   }
+
+  @Get(':instanceId')
+  @ApiOperation({summary: 'Get information about an instance.'})
+  @ApiResponse({description: 'The instance has successfully deleted.', status: 200})
+  @ApiResponse({description: 'Instance not found', status: 404})
+  @ApiResponse({description: 'Validation error', status: 422})
+  @ApiResponse({description: 'Internal server error', status: 500})
+  async getInstance(@Param('instanceId') instanceId: string, @Res() res: Response): Promise<void> {
```

Comment:
You can check with `validator.isUUID(instanceId, 4);` if the instance is a valid UUID v4.